network/src/peer/mod.rs

```diff
@@ -492,13 +507,11 @@ where
                     );
                 }
             }
-            PeerRequest::CloseConnection => {
-                self.close_connection(DisconnectReason::Requested).await;
-            }
+            PeerRequest::CloseConnection => self.close_connection(DisconnectReason::Requested),
         }
     }
 
-    async fn close_connection(&mut self, reason: DisconnectReason) {
+    fn close_connection(&mut self, reason: DisconnectReason) {
```

Comment:
Haha, an async function that didn't have to be.  I think though, this is another one of those refactors that could be in a separate commit, because it doesn't necessarily have to do with fuzzing.

Reply:
it's also a one-liner function, I'm against these :P